packages/neos-ui/src/Containers/LeftSideBar/NodeTree/index.js

```diff
@@ -123,6 +147,13 @@ export default class NodeTree extends PureComponent {
 
         return (
             <Tree className={classNames}>
+                <button
+                    onClick={this.handleCollapseAll}
+                    className={style.collapseAll}
+                    title={i18nRegistry.translate('collapseAll')}
```

Comment:
```suggestion
                    title={i18nRegistry.translate('Neos.Neos.Ui:Main:collapseAll')}
```

That solves it. Without the `Neos.Neos.Ui:Main:` prefix, the `i18nRegistry` assumes `Neos.Neos:Main:` (for historical reasons).